PublicationsRouter/v2/api/API.md

```diff
@@ -585,3 +586,52 @@ If the notification content is found and authentication succeeds you will receiv
     Binary Package
 
 Note that a successful access by a Repository account user will log a successful delivery of content notification into PubRouter (used for reporting on PubRouter's ability to support REF compliance).
+
+# API for matching configuration
+If you are a repository user you have access to the endpoint `/config`. This endpoint allows you to either update or retrieve your current matching configuration. This endpoint requires authentication, using an api key. 
```

Comment:
`If you have an Institution account you have access to the endpoint `/config` which allows you to retrieve and/or replace your current matching configuration.`

No comma needed in last sentence.